common/remote/sockfile.cpp

```diff
@@ -1052,7 +1086,7 @@ class CRemoteBase: public CInterface
                         WARNLOG("Remote file authenticate %s for %s ",e->errorMessage(err).str(),ep.getUrlStr(eps.clear()).str());
                         e->Release();
                         if (!retries)
-                            break;
+                            break; // MCK - is this a warning or an error ? If an error, should we close and throw here ?
```

Comment:
Btw, only other usage of connectSocket is in CRemoteBase::connect(), which is not used by anything. Should delete.